.github/workflows/smoke-test-release.yml

```diff
@@ -23,31 +23,31 @@ jobs:
             contents: read
         runs-on: ubuntu-20.04
         outputs:
-            tag: ${{ steps.tag.outputs.result }}
+            tag: ${{ steps.get-tag.outputs.tag }}
             created: ${{ steps.created-at.outputs.created }}
         steps:
             - name: Validate tag
               if: ${{ github.event_name == 'workflow_dispatch' }}
               env:
-                  GH_TOKEN: ${{ github.token }}
-              run: gh release view "${{ github.event.inputs.tag }}" --repo=woocommerce/woocommerce
+                  GH_TOKEN: ${{ secrets.E2E_GH_TOKEN }}
```

Comment:
As noted in the PR description, I replaced the default `GITHUB_TOKEN` with `E2E_GH_TOKEN`. You would see similar diffs like this throughout this PR.